src/core/Rule.ml

```diff
@@ -358,6 +358,23 @@ and extract_reduction = Separate | Concat [@@deriving show]
 and extract_transform = NoTransform | Unquote | ConcatJsonArray
 [@@deriving show]
 
+(*****************************************************************************)
+(* Secrets mode *)
+(*****************************************************************************)
+
+(*TODO: actualy figure this out*)
```

Comment:
What do you need to figure out?
Maybe be clearer in the comment on what is the TODO

Reply:
I think thats an old TODO; when I was drafting syntaxes; will delete that line